include/swift/SIL/SILInstruction.h

```diff
@@ -303,12 +356,12 @@ class SILInstruction
   }
 
   SILNode *getCanonicalSILNodeInObject() {
-    assert(isCanonicalSILNodeInObject() &&
+    assert(isRepresentativeSILNodeInObject() &&
            "the SILInstruction subobject is always canonical");
     return this;
   }
   const SILNode *getCanonicalSILNodeInObject() const {
-    assert(isCanonicalSILNodeInObject() &&
+    assert(isRepresentativeSILNodeInObject() &&
            "the SILInstruction subobject is always canonical");
     return this;
   }
```

Comment:
These have the wrong name now, if they're really needed at all.

Reply:
I was cargo-culting your design from other places. I will remove them.